### PR TITLE
Add aliases to bug report template

### PR DESCRIPTION
**Pull request by chitoku-k**

### CODE REVIEW

**Review by hexium310 — COMMENTED** [review]
How about that you add 'if you use an alias, write a command with the alias as it is' to 'What happened?'

**Comment by chitoku-k** [comment]
What is benefits compared to this template?

**Comment by hexium310** [comment]
It may prevent a reporter from using the tact to mention a command whose alias is removed.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -86,7 +97,7 @@ body:
       required: true
   - type: textarea
     attributes:
-      label: What happened?
+      label: What happened? Describe the command as it is despite any usage of alias.
```
Comment (hexium310) [inline .github/ISSUE_TEMPLATE/bug.yml:100]: I think you might want to use not `label` but `value`.

Comment (chitoku-k) [reply .github/ISSUE_TEMPLATE/bug.yml:100]: `value` is way smaller... OK

**Review by hexium310 — APPROVED** [review]

**Pull request merged**